csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,279 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: 'job!=""',
+    groupLabels: ['job', 'project_id'],
+    instanceLabels: [],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization',
+    },
+    signals: {
+      instanceCount: {
+        name: 'Instance count',
+        description: 'Number of VM instances',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'count(sum by (instance_name) (stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}))',
+            legendCustomTemplate: '',
+          },
+        },
```

Comment:
Here you could use `type='gauge'`,  `exprWrappers: [['count(', ')']],` `expr='stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}'`.

Reply:
I will add the count in the `exprWrappers `, but keep the raw type because I want to do the `sum by` only by `instance_name`, otherwise it will not return the sum uniquely counted by the unique name, but by job which is the same for all